spyderlib/utils/introspection/jedi_plugin.py

```diff
@@ -54,10 +54,9 @@ def load_plugin(self):
         self._warmup_thread.start()
 
     def get_completions(self, info):
-        """Return a list of completion strings"""
+        """Return a list of (completion, type) tuples"""
         completions = self.get_jedi_object('completions', info)
-        debug_print(str(completions)[:100])
```

Comment:
Do you plan to remove or leave this `debug_print`? You haven't answered my question :-)